usbinfo/darwin.py

```diff
@@ -97,7 +101,7 @@ def _disk_partition(node):
     info = {}
 
     for child in node.get('IORegistryEntryChildren', []):
-        for class_name, handler in ioclass_handlers.iteritems():
+        for class_name, handler in ioclass_handlers.items():
```

Comment:
As per [PEP 569](https://www.python.org/dev/peps/pep-0469/) this should be changed to `iter(ioclass_handlers.items())`